### ISSUE TITLE
EntityNotes->getOne возвращает NoteModel , игнорируя note_type и params

### ISSUE THREAD

**Issue by dura4kov** [amocrm/amocrm-api-php#6]
EnitiyNotes->getOne возвращает объекта класса NoteModel независимо от принадлежности примечания тому или иному типу.
https://github.com/amocrm/amocrm-api-php/blob/46952a8682b54bf56e74a41bd82f38f66bd0ea26/src/AmoCRM/EntitiesServices/Traits/WithParentEntityMethodsTrait.php#L71-L79

Хотелось бы, чтоб EnitiyNotes->getOne работал по аналогии с CustomFields->getOne - объект создавался с помощью соотв. фабрики (NoteFactory::createForType)
https://github.com/amocrm/amocrm-api-php/blob/46952a8682b54bf56e74a41bd82f38f66bd0ea26/src/AmoCRM/EntitiesServices/CustomFields.php#L299-L303

**Comment by bessudnov** [comment]
Согласен, было бы удобнее, думаю до середины июня будет время допилить
